src/services/reminder-service.ts

```diff
@@ -24,9 +25,8 @@ export class ReminderService {
 
   static sendGuildReminder(reminder: GuildReminder, client: Client): void {
     try {
-
       (client.channels.resolve(reminder.target.channel) as TextChannel)
-        .send(reminder.content);
+        .send(reminder.content || 'reminder');
```

Comment:
Just 'reminder' seems a bit bland maybe?

Reply:
suggest something else.
maybe time of reminder

Reply:
Mention the user with 'here's your reminder for {time}.' or something like that.